### ISSUE TITLE
[RFCxxxx_Lax] Handling multiple ocurrences of the same BYxxx rule

### ISSUE THREAD

**Issue by yahrens** [dmfs/lib-recur#14]
Example:

```
"FREQ=YEARLY;BYMONTH=2;BYMONTH=3;BYMONTH=4"
```

We have several options here.
1. Throw an exception (probably nonsense since we're in Lax mode)
2. Either pick the first or the last one, ie. "BYMONTH=2" or "BYMONTH=4"
3. Merge them together into one list, ie. "BYMONTH=2,3,4"

**Comment by ekubovsky** [comment]
Option #3 makes sense in LAX mode

**Comment by dmfs** [comment]
True, but I'm hesitating to implement this until I've actually seen such a broken rule in the wild.
